app/controllers/post.js

```diff
@@ -1,16 +1,89 @@
 const userModel = require('../models/User')
 const bcrypt = require('bcrypt')
 const jwt = require('jsonwebtoken')
+const PostModel = require('../models/Post')
+const HANDLER = require('../utils/response-helper')
+const STATUS = require('../utils/status-codes')
+const helper = require('../utils/uploader')
```

Comment:
make the constant name as `imgUploadHelper`

Reply:
Sure :)